crates/ika-archival/src/lib.rs

```diff
@@ -97,7 +99,8 @@ const MANIFEST_FILENAME: &str = "MANIFEST";
 )]
 #[repr(u8)]
 pub enum FileType {
-    CheckpointMessage = 0,
+    IkaSystemCheckpoint = 0,
+    CheckpointMessage = 1,
 }
```

Comment:
```suggestion
    IkaSystemCheckpoint = 0,
    CheckpointMessage = 1,
}
```
SystemCheckpointMessage
DWalletCoodinatorCheckpointMessage